case_studies/project_cipher/2015-10-25-meeting/Project Cipher (Jevin) - Sun 10:25

```diff
@@ -0,0 +1,138 @@
+Project Cipher (Jevin) - Sun 10/25
+
+Hosted first formal workshop meeting yesterday
+
+Did a custom-version of jon’s portfolio workshops
+
+—added a bit more stuff to it for more depth
```

Comment:
not sure the best way to fix it so I deleted - and redid them all with - in sublime

I'm using the Mac Notes app.